tests/errors/StratNoPropRec.hs

```diff
@@ -0,0 +1,14 @@
+{-@ LIQUID "--expect-error-containing=The constructor Test.VArr of the type Test.Val was declared stratified but it has a recursive occurence of type Test.Val which is not a Prop type" @-}
```

Comment:
The new test modules need to be listed in `tests.cabal` to get them built together with the other tests.